pkg/controller/useraccount/useraccount_controller.go

```diff
@@ -118,55 +118,54 @@ func (r *ReconcileUserAccount) Reconcile(request reconcile.Request) (reconcile.R
 	// If the UserAccount has not been deleted, create or update user and identity resources.
 	// If the UserAccount has been deleted, delete secondary resources identity and user.
 	if !util.IsBeingDeleted(userAcc) {
-		reqLogger.Info("Adding finalizer on UserAccount")
 		// Add the finalizer if it is not present
-		if err := r.addFinalizer(userAcc); err != nil {
+		if err := r.addFinalizer(logger, userAcc); err != nil {
 			return reconcile.Result{}, err
 		}
 	}
 	if !util.IsBeingDeleted(userAcc) && !userAcc.Spec.Disabled {
-		reqLogger.Info("Ensuring user and identity associated with UserAccount")
+		logger.Info("ensuring user and identity associated with UserAccount")
```

Comment:
Just curios why you changed the upper case to the lower case in some log messages?

Reply:
for the sake of consistency: most of the messages start with a lower case. But to be honest I'm not sure what's our policy with regards to log messages